servers/src/main/java/tachyon/master/block/BlockMasterWorkerServiceHandler.java

```diff
@@ -31,6 +32,7 @@
 /**
  * This class is a Thrift handler for block master RPCs invoked by a Tachyon worker.
  */
+@ThreadSafe
```

Comment:
why is this class threadsafe?
It seems to me it is a wrapper of a `BlockMaster` instance, which is not thread-safe.